src/prefect/cli/profile.py

```diff
@@ -251,6 +252,17 @@ def rename(name: str, new_name: str):
     profiles.add_profile(profiles[name].copy(update={"name": new_name}))
     profiles.remove_profile(name)
 
+    # Switch the active profile to the new name if provided
```

Comment:
I find this comment a bit confusing, maybe:

```suggestion
    # If the active profile was renamed switch the active profile to the new name.
```

?

Reply:
Yeah idk what I was typing there, thanks